waku/v2/peermanager/peer_manager.go

```diff
@@ -31,9 +37,12 @@ type PeerManager struct {
 	serviceSlots        *ServiceSlots
 	ctx                 context.Context
 	sub                 event.Subscription
+	topicMutex          sync.RWMutex
+	subRelayTopics      map[string]*NodeTopicDetails
 }
 
 const peerConnectivityLoopSecs = 15
+const relayOptimalPeersPerShard = 6
```

Comment:
I wonder if it's possible to obtain this value from wakurelay.
It should be possible to obtain `Relay().Params().D`, the problem is how to inject this value to the peer manager.

Reply:
Yeah, this should be from configuration.
Since we set value of D also from configuration in relay, maybe we can pass the same during PeerManager initialization.